extendr-api/tests/scalar_tests.rs

```diff
@@ -49,6 +49,69 @@ fn test_rint() {
     assert!((a + b).is_na());
 }
 
+#[test]
+fn test_rint_opassign() {
+    // LHS Rint, RHS Rint
+    let mut a = Rint::from(20);
+    a += Rint::from(10);  assert_eq!(a, Rint::from(30));
+    a -= Rint::from(20);  assert_eq!(a, Rint::from(10));
+    a *= Rint::from(20);  assert_eq!(a, Rint::from(200));
+    a /= Rint::from(100); assert_eq!(a, Rint::from(2));
```

Comment:
I am not a fan of multiple statements per one code line.
If `cargo fmt` allows this, then I am OK.